pkg/nsx/util/namespace.go

```diff
@@ -0,0 +1,67 @@
+/* Copyright © 2021 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Not sure if we need copyright header. If needed may be use the latest one like
```
/* Copyright © 2025 Broadcom, Inc. All Rights Reserved.
   SPDX-License-Identifier: Apache-2.0 */
```

Reply:
ok